testsuite/validation/input/pot3d.dat

```diff
@@ -1,30 +1,29 @@
- &topology
-  nr=63
-  nt=91
-  np=225
- /
- &inputvars
-  ifprec=1
-  option='ss'
-  r1=2.5
-  rfrac=0.0,1.0
-  drratio=2.5
-  nfrmesh=0
-  tfrac=0.00
-  dtratio=1.0
-  nftmesh=0
-  pfrac=0.00
-  dpratio=1.0
-  nfpmesh=0
-  phishift=0.
-  br0file='br_input_tiny_bin.bin'
-  phifile=''
-  brfile=''
-  btfile=''
-  bpfile=''
-  br_photo_file=''
-  ncghist=10
-  ncgmax=10000000 
-  epscg=1.e-12
-  idebug=0
- /
+# Topology Configuration
```

Comment:
I'll need to fix something's to ensure that we don't need any changes in pot3d.dat file at all.

Reply:
Done, I've fixed this.